src/styles/card.scss

```
@@ -0,0 +1,105 @@
+:root {
+  font-family: $font;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  width: 200px;
+  height: 408px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid $color-border;
+  border-radius: 5px;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__container {
+    @include spaced-center;
+  }
+
+  &__info {
+    margin-top: 40px;
+  }
+
+  &__name {
+    margin: 0;
+    font-size: 12px;
+    font-weight: 500px;
```

font-weight value is not pixels - fix all cases
```suggestion
    font-weight: 500;
```